frontend/src/configureStore.tsx

```diff
@@ -2,10 +2,11 @@ import { createStore, applyMiddleware } from 'redux';
 import logger from 'redux-logger';
 import thunk from 'redux-thunk';
 import { rootReducer } from 'reducers/rootReducer';
+import { loadingBarMiddleware } from 'react-redux-loading-bar';
 
 export default function configureStore() {
   if (process.env.NODE_ENV === 'development') {
-    return createStore(rootReducer, applyMiddleware(thunk, logger));
+    return createStore(rootReducer, applyMiddleware(thunk, logger, loadingBarMiddleware()));
```

Comment:
just curious what does this do?

Reply:
It's a pre-built slice for managing the loading bar.  You can see a blue line at the top loading.